guides/fcm-flutter-novu/how-to-send-push-notifications-to-flutter-apps-with-fcm-using-Novu.mdx

```diff
@@ -0,0 +1,735 @@
+---
+title: 'How to send push notifications to Flutter apps (Android & iOS) with FCM using Novu'
+description: 'Learn how to integrate Firebase Cloud Messaging with Novu and send notifications to Flutter apps'
+icon: 'circle-8'
+---
+
+## Prerequisites
+
+To complete this guide, you will need the following:
+
+- [Apple Developer](https://developer.apple.com/) membership (to obtain the required permissions to send push notifications).
+- A machine running MacOS to work on building the Flutter app for iOS devices.
+- [Firebase](https://firebase.google.com/) account
+- [Novu](https://web.novu.co/) account.
+- [Android Studio](https://developer.android.com/studio/install) configured with Dart and Flutter plugins.
+- [Xcode](https://developer.apple.com/xcode/) installed on your machine.
+
+Check out the [Flutter app code for this guide on GitHub](https://github.com/novuhq/flutter-fcm-novu).
+
+## Set up and Create a Flutter App
+
+If you have an existing Flutter App, you can skip this step.
+
+If you are creating a brand new Flutter app for the first time, please follow this [guide](https://firebase.google.com/codelabs/firebase-fcm-flutter#0).
```

Comment:
I feel that "for the first time" can be removed here as it makes it sound as if this is the very first flutter app one is creating

Reply:
Yeah, the reason I added it is if the reader has never created a new flutter app ever before. But I will remove it.